secboot/secboot_sb.go

```diff
@@ -177,7 +177,15 @@ func UnlockEncryptedVolumeWithRecoveryKey(name, device string) error {
 		KeyringPrefix:    keyringPrefix,
 	}
 
-	if err := sbActivateVolumeWithRecoveryKey(name, device, nil, &options); err != nil {
+	authRequestor, err := sb.NewSystemdAuthRequestor(
```

Comment:
The auth requestor is built in different places with the same arguments, probably this could be shared by using a helper function.

Reply:
Done.